services/apps/alcs/src/alcs/notice-of-intent-decision/notice-of-intent-decision.entity.ts

```diff
@@ -0,0 +1,85 @@
+import { AutoMap } from '@automapper/classes';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  Index,
+  ManyToOne,
+  OneToMany,
+} from 'typeorm';
+import { Base } from '../../common/entities/base.entity';
+import { NoticeOfIntent } from '../notice-of-intent/notice-of-intent.entity';
+import { NoticeOfIntentDecisionDocument } from './notice-of-intent-decision-document/notice-of-intent-decision-document.entity';
+import { NoticeOfIntentDecisionOutcome } from './notice-of-intent-decision-outcome.entity';
+
+@Entity()
+@Index(['resolutionNumber', 'resolutionYear'], {
+  unique: true,
+  where: '"audit_deleted_date_at" is null and "resolution_number" is not null',
+})
+export class NoticeOfIntentDecision extends Base {
+  constructor(data?: Partial<NoticeOfIntentDecision>) {
+    super();
+    if (data) {
+      Object.assign(this, data);
+    }
+  }
+
+  @AutoMap()
+  @Column({ type: 'timestamptz' })
+  date: Date;
+
+  @AutoMap()
+  @Column({ type: 'timestamptz', nullable: true })
+  auditDate: Date | null;
+
+  @AutoMap(() => NoticeOfIntentDecisionOutcome)
+  @ManyToOne(() => NoticeOfIntentDecisionOutcome, {
+    nullable: false,
+  })
+  outcome: NoticeOfIntentDecisionOutcome;
+
+  @AutoMap()
+  @Column()
+  outcomeCode: string;
```

Comment:
no column descriptions 🥲

Reply:
I have a feeling we won't be going back and adding them anytime soon